packages/algolia-search/components/SearchBox/SearchBox.tsx

```diff
@@ -51,7 +51,7 @@ export function SearchBox(props: SearchBoxProps) {
       inputRef={searchInputElement}
       onChange={debounceSearch}
       fullWidth
-      sx={{ mt: 1, mb: 1 }}
+      sx={[{ mt: 1, mb: 1 }, ...(Array.isArray(sx) ? sx : [sx])]}
```

Comment:
Are the margins still relevant after using the `CategorySearchResults` component on the search page?

Reply:
We still want this margin if there are no search results so I would say yes